multichain-testing/scripts/noble-usdn-lab.ts

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env -S node --import ts-blank-space/register
+/**
+ * @file verify that `nobled tx swap swap ...` etc. work
+ *
+ * Usage:
+ *
+ * To add USDC to trader1's account on Noble:
+ *   FAUCET=1 noble-usdn-lab.ts
+ *
+ * To transfer 123uusdc from Noble to Agoric:
+ *   TXFR=123 [NET=testnet] noble-usdn-lab.ts
+ *
+ * To create a stableswap pool in starship:
+ *   POOL=1 noble-usdn-lab.ts
```

Comment:
@Muneeb147 I don't expec this POOL thing to work; this was my _attempt_ to get it to work before I learned about `nobled tx authority execute ...` (cf. #11513)

SWAP and LOCK are the main targets.